### PR TITLE
Add script for test report and Add command of stopping spec's test on GUI

### PR DESCRIPTION
**Pull request by DaiYuzeng**
There are two new features:

- Generate report
- Stop failed test on GUI via new command `stopOnFailed`

### Generate report

`yarn run report` can generate report now and the report will be saved as `mochawesome-report/mochawesome.html`

The content of HTML looks like:
image

There are records about Test status/Error message/Spent times of test.

### Stop test

Dependent tests should be stopped if it's failed.
Use code below to stop it.

```
before(() => {
    cy.stopOnFailed();
})
```

### CODE REVIEW

**Comment by WuJun2016** [comment]
@DaiYuzeng   For a description of how to use `cy.stopFailed` please add it to `cypress/README.md`  （You can wait until the pr is merged before adding  https://github.com/harvester/tests/pull/343）, thanks

File: cypress/package.json
Comment (lanfon72) [inline cypress/package.json:29]: `yarn` is not required by default, we can use `npm run delete:reports` instead.

Comment (DaiYuzeng) [reply cypress/package.json]: thanks!

**Review by WuJun2016 — APPROVED** [review]
LGTM

**Pull request merged**
